targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F412xG/TARGET_MTB_MXCHIP_EMW3166/PinNames.h

```diff
@@ -201,6 +201,7 @@ typedef enum {
     // Generic signals namings
     LED1        = PB_2,
     LED2        = PB_10,
+    LED3        = NC,
```

Comment:
@KariHaapalehto : I'm not clear why is this required?

Reply:
It is not required, but all MTB's do have 3 different led's so I would like to identyfy that there is 3rd led available but it is not connected.